lucida/commandcenter/controllers/Config.py

```diff
@@ -12,29 +13,45 @@
 
 # Pre-configured services.
 # The ThriftClient assumes that the following services are running.
-# Host IP addresses are resolved dynamically: 
+# Host IP addresses are resolved dynamically:
 # either set by Kubernetes or localhost.
-SERVICES = { 
-	'IMM' : Service('IMM', 8082, 'image', 'image'), 
-	'QA' : Service('QA', 8083, 'text', 'text'),
-	'CA' : Service('CA', 8084, 'text', None),
-	'IMC' : Service('IMC', 8085, 'image', None),
-	'FACE' : Service('FACE', 8086, 'image', None),
-	'DIG' : Service('DIG', 8087, 'image', None),
-	'ENSEMBLE' : Service('ENSEMBLE', 9090, 'text', None) 
-	}
+SERVICES = {
+    'IMM' : Service('IMM', 8082, 'image', 'image'),
+    'QA' : Service('QA', 8083, 'text', 'text'),
+    'CA' : Service('CA', 8084, 'text', None),
+    'IMC' : Service('IMC', 8085, 'image', None),
+    'FACE' : Service('FACE', 8086, 'image', None),
+    'DIG' : Service('DIG', 8087, 'image', None),
+    'ENSEMBLE' : Service('ENSEMBLE', 9090, 'text', None),
```

Comment:
I'm not familiar with ENSEMBLE. It is not a service that is being started through the Makefile so I'm assuming it is not doing anything. This is the only documentation I can find for ENSEMBLE on the top README: _There is another service called "ENSEMBLE" (ensemble-style question answering) which is not part of this repo. We will provide instructions on how to set it up soon._ It would make sense to remove it, right?